lib/pokerscore/set.rb

```diff
@@ -0,0 +1,23 @@
+class Set
+  attr_reader :value, :length
+  def initialize(value, length)
```

Comment:
Huge nitpick (I might even be wrong to say this), but I believe we generally like to keep space around our initialize method (unless we're adding a doc string).